HIPs/hedera-hashgraph.md

```diff
@@ -0,0 +1,318 @@
+We propose to host a clone of that at https://NAME.PROJECT.com. 
+
+## Build
+As you can see in this chapter, `PROJECT` contains many sub-projects that need specific build steps and infrastructure.
+Therefore, it is important to invest in `PROJECT`’s build, release, and deployment.
+The goal must be to have a vendor-neutral build and release process led by LF.
+We propose to nominate a release manager by LF who is involved in the project’s transition as soon as possible.
+
+## Slack
+With all the given sub-projects and the big community around `PROJECT`, it will be a must to have an open and easy way
+to communicate with the community.
+Therefore, we propose to create a `PROJECT` Slack.
+
+## GitHub applications
+We have several GitHub applications that are used in the repositories today.
+We propose to migrate those applications with the repositories and check, as a step in the near future,
+what can be replaced by LF best practices.
+
+# How To
+All repositories that should be transitioned to `PROJECT` have already defined an entire CI pipeline today.
+As already mentioned, GitHub actions are used to run builds and tests for each pull request / commit in the projects.
+Since the GitHub actions are part of the repositories the full build and test workflow of all projects is
+already open today.
+Since some projects have significantly more complex workflows regarding build and testing we will give some additional insights.
+
+All projects are based on standards and best practices (Maven, Gradle, NPM, …) regarding the build process.
+Based on that, each project can easily be built on a local machine.
+We support Linux, macOS, and Windows as environments for most projects.
+Only projects that already have a specific need based on their dependencies or programming language are more restricted.
+An example is the Swift SDK, which needs macOS to be built.
+
+The projects differ greatly in deployment and hosting, so `PROJECT` will contain libraries that are only
+used as 3rd party dependencies in external projects next to full backend services and web applications.
+Each project type is described in the sections below.
+
+All projects provide GitHub actions for release automation.
+That automation includes tagging the project, signing artifacts, and creating release notes.
+Some of those steps need specific and secret environment variables like GitHub tokens.
+All the tokens need to be created for the new repositories to support the release of the software.
+
+## SDKs
+All SDK projects and some other repositories provide workflows to deploy the build artifacts of a release to a global
+registry, like Maven Central or NPM, which require accounts and tokens.
+Here we propose to create new accounts and tokens in all needed registries to upload all releases done under `PROJECT`
+in new namespaces of `PROJECT`.
+
+## Consensus Node
+The Consensus Node project defines the software that is running on each node of a network.
```

Comment:
I didn't see a Consensus Node project in the list of github repositories above.
Is this https://github.com/hashgraph/hedera-services? If so it would be helpful to indicate this in the github repository description above.

Reply:
yes, you are right. "Hedera-services" contains the code of a consensus-node and i personally hope that we can name it better under LF ;)